cfgov/v1/__init__.py

```diff
@@ -73,11 +73,11 @@ def parse_links(soup):
             pass
 
     for a in soup.find_all('a', href=True):
-        # Sets the link to an external one if you're leaving .gov 
-        if extlink_pattern.match(a['href']):
-            a['href'] = '/external-site/?ext_url=' + a['href']
         # Sets the icon to indicate you're leaving consumerfinance.gov
         if noncfpb_pattern.match(a['href']):
+            # Sets the link to an external one if you're leaving .gov 
+            if extlink_pattern.match(a['href']):
+                a['href'] = '/external-site/?ext_url=' + a['href']
```

Comment:
Nope, it was just setting the link. The icon was being set under a separate conditional so it never got hit for `extlink_pattern.match(a['href'])`